plugin/health/README.md

```diff
@@ -48,9 +49,31 @@ net {
 }
 ~~~
 
+~~~ corefile
+com {
+    whoami
+    health
+}
+
+net {
+    erratic
+    health
+}
+~~~
+
+ 
+~~~ corefile
+com org net {
+    whoami
+    health
+}
+~~~
+
 ## Plugins
 
 Any plugin that implements the Healther interface will be used to report health.
+When the same port is reused, all the instances of the all the plugins that are associated with "health" 
+will participate to the result
```

Comment:
I think I know what you are trying to say, but doesn't seem to come through in this sentence.

```
Each health plugin will export the health of all the plugins (that export health) in the server. If ports are shared, multiple server plugins will be included as well.
```
or something like that.